ISSUE_REQUEST.md

```diff
@@ -0,0 +1,5 @@
+To help us diagnose issues efficiently, please include:
+
+- A short but descriptive title
+- A detailed description of the problem including relevant software versions and steps to reproduce
```

Comment:
nit. Can we add a '.' at the end of each sentence?

Reply:
Hi!  I have been using the practice of only adding periods when lists contain full sentences.  That's standard practice for business writing (see http://www.businesswritingblog.com/business_writing/2012/01/punctuating-bullet-points-.html for an example)